src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -6940,6 +6925,57 @@ namespace PackagedTerminalHeatPump {
 
 	}
 
+	void
+	SetMinOATCompressor(
+		int const PTUnitNum, // index to furnace
+		std::string const PTUnitName, // name of furnace
+		std::string const cCurrentModuleObject, // type of furnace
+		std::string const CoolingCoilType, // type of cooling coil
+		std::string const CoolingCoilName, // name of cooling coil
+		std::string const HeatingCoilType, // type of heating coil
+		std::string const HeatingCoilName, // name of heating coil
+		bool & ErrorsFound // GetInput logical that errors were found
+	) {
```

Comment:
This function looks like it is called once, so I am less concerned about the strings.  If any more changes are made we should go ahead and make them pass by reference.